### PR TITLE
freeze PT 1.5 for Horovod issue

### PR DESCRIPTION
**Pull request by Borda**
## What does this PR do?

temporal freeze for Horovod incompatibility with pt 1.6

## PR review    
Anyone in the community is free to review the PR once the tests have passed.     
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

**Comment by Borda** [comment]
@tgaddair mind have look why Horovod is failing with PyTorch 1.6

File: requirements/base.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # the default package dependencies
 
 numpy>=1.16.4
-torch>=1.3
+torch>=1.3, <1.6  # TODO: temporary freze fro Horowon incomaptibility with 1.6
```
Comment (yukw777) [inline requirements/base.txt:4]: typo: `temporary freeze for Horovod incomaptibility with 1.6`

Comment (Borda) [reply requirements/base.txt:4]: mind use suggestions? ;]

**Comment by Borda** [comment]
@awaelchli any idea about this docs build failing?
```
Warning, treated as error:
failed to reach any of the inventories with the following issues:
intersphinx inventory 'https://pytorch.org/docs/stable/objects.inv' not fetchable due to <class 'requests.exceptions.HTTPError'>: 404 Client Error: Not Found for url: https://pytorch.org/docs/stable/objects.inv
```

**Pull request merged**